lib/postmark/error.rb

```diff
@@ -72,8 +72,8 @@ class InactiveRecipientError < ApiInputError
     attr_reader :recipients
 
     PATTERNS = [/^Found inactive addresses: (.+?)\.$/.freeze,
-                /^Found inactive addresses: (.+?)\.$/.freeze,
-                /these inactive addresses: (.+?)\. Inactive/.freeze].freeze
+                /these inactive addresses: (.+?)\. Inactive/.freeze,
+                /these inactive addresses: (.+?)\.{0,1}$/].freeze
```

Comment:
```suggestion
                /these inactive addresses: (.+?)\.?$/].freeze
```
We could just use the question mark to match `.`  zero or one time, right?

Reply:
yep ? works good to hombre!